known-issues.yml

```diff
@@ -730,6 +730,15 @@ zib-VisualFunction.VisualAid.Product:
     Device.udiCarrier:*UdiCarrier.carrierHRF:
       - datatype: string instead of CD compatible datatype
         reason: The mapping of ProductID is not straightforward and relates to multiple elements in FHIR. In result, the mapping of the concept to FHIR will not match on a datatype level.
+        
+zib-wounds.WoundImage:
+  zib deviations:
+    DocumentReference.content.attachment.url:
+      - alias: WondFoto instead of Wondfoto
+        for: NL-CM:19.4.11 (Burnwound.WoundImage)
```

Comment:
There still is a QA error present regarding this deviation. Should the known issue be formulated somewhat differently?

Reply:
@pieter-edelman-nictiz will look into it

Reply:
Fixed by @pieter-edelman-nictiz